### ISSUE TITLE
Repository maintenance

### ISSUE THREAD

**Issue by nraw** [esamattis/slimux#86]
It seems like this repository has not been updated for the last 4 years, even though there have been pull requests throughout this time.

To me personally, this repo represents the core of my workflow, so I would love to see it continue being maintained.

@esamattis any vision on potential further development or should there be an official fork that continues development?

**Comment by esamattis** [comment]
Yeah, I don't personally use this anymore so cannot really dedicate time to maintain this actively. If somebody wants to start a fork that merges the PRs from here I'm happy to link to it from the README.

Edit: I linked this issue https://github.com/esamattis/slimux/commit/ebc0ef3164c07b00e0b63113cf627637af424352

**Comment by LinuxIsCool** [comment]
Someone please maintain this repo it's such a valuable tool to so many people :pray:
